views/includes/check-your-answers/beneficial-owner-gov.html

```diff
@@ -3,8 +3,12 @@
 {% set bogStartDate = dateMacros.formatDate(bog.start_date["day"], bog.start_date["month"], bog.start_date["year"]) %}
 
 {% set bogFormattedPrincipalAddressHtml %}
-  {% set address = bog.principal_address %}
-  {% include "includes/display_address.html" %}
+  {% if (not bog.principal_address|length > 0  and pageParams.noChangeFlag) %}
```

Comment:
These probably read better the other way around, at first glance it looks like NOT applies to both variables... but it still works so I won'y delay things :-)

Reply:
Hahahaha okay, took out the not and compared the length to 0.